src/main/java/com/example/bookstoreapp/dto/shoppingcart/CartItemUpdateDto.java

```diff
@@ -0,0 +1,10 @@
+package com.example.bookstoreapp.dto.shoppingcart;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record CartItemUpdateDto(
```

Comment:
A little bit confusing naming. Is it request or Response? 
CartItemUpdateRequestDto